File: commands/echo.js
module.exports = {
  name: 'echo',
  aliases: ['say'],
  description: 'Repeat what is said',
  usage: 'Hello.',
  execute(message, args) {
    const fArg = message.content.substr(message.content.indexOf(' ') + 1);
      if (message.member.id == 85164966612570112){
        if (message.member.permissions.has('MANAGE_MESSAGES')) {
          message.channel.send(fArg);
          message.delete({ timeout: 5000, reason: 'For echo' });
        }
        else {
          message.channel.send('"' + fArg + '"');
        }
    }
  },
};

File: Deprecate/discord_bot/commands/echo.py
name = 'echo'
aliases = ['say']
description = 'Repeat what is said'
usage = 'Hello.'

async def execute(client, message, args):
    fArg = message.content[len(message.content.split()[0]):].strip()  # Get the argument after the command
    await message.channel.send(fArg)
